### PR TITLE
Fix Disk tab in advanced view at Hebrew show as mirror view.

### PR DESCRIPTION
**Pull request by talregev**
Fix: https://github.com/BOINC/boinc/issues/3822 https://github.com/BOINC/boinc/issues/988

I try to look on the CPP code, and understand where is the decision is made that the disk tab is mirror when choosing Hebrew.
All other tab is nor mirror, it specific tab, so it made me believe that the decision is in our code.
Any help will be appreciated, hopefully I will crush this bug.

Can you help me find some pointer in the cpp code? Thanks in advance.

### CODE REVIEW

**Comment by talregev** [comment]
I force the layout to be LTR in the disk tab and the statistics tab, I change to Hebrew, but I cannot see the Hebrew words.  
Fail to load somehow the language. I guess it cannot find the language files. 
Also the boinc cannot connect to the internet. I am not sure why. 
Any Hint? 
I compile with visual studio community 2019, on windows 10, running boinc manager inside visual studio.

**Comment by AenBleidd** [comment]
Not sure about the Internet, but if you want locale to be used in Manager, you should copy 'locale' folder from the installation location to your debug executable folder

**Comment by talregev** [comment]
Thanks @AenBleidd It was help!
I have disk tab results: 
The hebrew now is not mirror although all sentence is to the left. but it better then when it mirror. it improvement! 
I cannot verify not the statistics tab because it lacking to connect to the internet.  

Before
After
Hebrew after

**Comment by talregev** [comment]
Also fix the statistics now it look perfect :)
Statistics after

File: clientgui/ViewStatistics.cpp [hunk 1]
```diff
@@ -1945,6 +1945,8 @@ CViewStatistics::CViewStatistics(wxNotebook* pNotebook) :
 	m_PaintStatistics = new CPaintStatistics(this, ID_LIST_STATISTICSVIEW, wxDefaultPosition, wxSize(-1, -1), 0);
 	wxASSERT(m_PaintStatistics);
 
+	m_pTaskPane->SetLayoutDirection(wxLayout_LeftToRight);
```
Comment (AenBleidd) [inline clientgui/ViewStatistics.cpp:1948]: Could you please fix indents it this whole funtion?

**Review by AenBleidd — CHANGES_REQUESTED** [review]

File: clientgui/ViewStatistics.cpp (hunk 1 above)
Comment (talregev) [reply clientgui/ViewStatistics.cpp:1948]: Yes, I did

**Pull request merged**